Packs/Reco/Integrations/Reco/Reco.py

```diff
@@ -474,12 +581,45 @@ def parse_table_row_to_dict(alert: List[Dict[str, Any]]) -> Dict[str, Any]:
             if parsed_time:
                 obj[key] = parsed_time.strftime(DEMISTO_OCCURRED_FORMAT)
         if key == "risk_level":
-            obj[key] = int(obj[key])
+            try:
+                obj[key] = int(obj[key])
+            except Exception:
+                obj[key] = str(obj[key])
```

Comment:
Why is the expression in the exception handling (under `except`) is the same as the expression under `try`?